.config/forest.dic

```diff
@@ -27,28 +35,34 @@ JWT
 Kademlia
 libp2p
 localhost
+mainnet
 MDNS
 mempool
 Merkle
+MiB
 multisig
 mutex
 P2P
 pointer/SM
 PoSt
 RPC
+RRS
```

Comment:
What is `RRS` and where is it used?

Reply:
Oh, in the mem tracker.

Reply:
I think that is a typo. It should be RSS (resident set size). And I'd prefer if we wrote out the abbreviation (because I always forget what it means).